presto-main/src/main/java/com/facebook/presto/connector/ConnectorContextInstance.java

```diff
@@ -28,17 +29,20 @@
     private final TypeManager typeManager;
     private final PageSorter pageSorter;
     private final PageIndexerFactory pageIndexerFactory;
+    private final DomainTranslator domainTranslator;
 
     public ConnectorContextInstance(
             NodeManager nodeManager,
             TypeManager typeManager,
             PageSorter pageSorter,
-            PageIndexerFactory pageIndexerFactory)
+            PageIndexerFactory pageIndexerFactory,
+            DomainTranslator domainTranslator)
```

Comment:
Given we likely will have multiple utilities for RowExpression, we probably should wrap it around in a RowExpressionService and use getDomainTranslator from it?